react/src/typings.tsx

```diff
@@ -250,3 +250,29 @@ export interface Option {
     disabled?: boolean;
     selected?: boolean;
 }
+
+export interface Gene {
+    gene_id: number;
+    hgnc_gene_id: number;
+    ensembl_id: number;
+    gene: string;
+    hgnc_gene_name: string;
+    variants?: Variant[];
+}
+
+export interface Variant {
+    variant_id: number;
+    analysis_id: number;
+    position: string;
+    reference_allele: string;
+    alt_allele: string;
+    variation: string;
+    refseq_change: string;
```

Comment:
```suggestion
    refseq_change?: string;
```

Reply:
updated in [bec4861](https://github.com/ccmbioinfo/stager/commit/bec48617015303e9dc4870e8a56d498c8052d646)